ontology/obo/testdata/mannosyltransferaseActivity.dot

```diff
@@ -0,0 +1,12 @@
+digraph G{
```

Comment:
I think the 3 test out files could be removed?
Otherwise, waiting for new commit before merging